tests/publiccloud/check_registercloudguest.pm

```diff
@@ -148,6 +148,50 @@ sub check_instance_unregistered {
     }
 }
 
+sub new_registration {
+    my ($instance) = @_;
+    record_info('Starting registration...');
+    $instance->ssh_script_retry(cmd => "sudo ${path}/registercloudguest $regcode_param", timeout => 300, retry => 3, delay => 120);
+    check_instance_registered($instance);
+
+    record_info('Testing docker on the instance.');
```

Comment:
hm and what code which testing docker and podman doing inside function with name `new_registration` ? :thinking:

Reply:
https://github.com/SUSE-Enceladus/cloud-regionsrv-client/blob/master/integration_test-process.txt#L38C1-L59C29  I will check with the owner why this is necessary :)

Reply:
I didn't question necessity of checking docker and podman services during checking registration , as you correctly stated this was developers choice and we should blindly follow it . 
I state pure code organization question - this function raise 2 concerns from my side : 
1. extracting a function which contains two actions which has very weak relations to each other . I would do one of two things - or not create this function at all and just keep all code in `run` or create TWO functions one for registration another for services check
2. `new_registration` name does not describe content inside

Reply:
Added a separate func for the containers.